xc7/archs/zynq7/devices/xc7z010-roi-virt/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project_xray_arch(
```

Comment:
Put this behind a function, and take the DEVICE and ROI_DIR argument rather using scoped variables

Reply:
Fixed